Makefile

```diff
@@ -72,18 +74,19 @@ po-pull:
 install-po-files:
 	$(MAKE) -C po install
 
-test:
-	@echo "***Running pylint checks***"
-	@find . -name '*.py' -print|xargs -n1 --max-procs=$(NUM_PROCS) pylint -E 2> /dev/null
-	@echo "[ OK ]"
-	@echo "***Running unittests checks***"
-	@PYTHONPATH=. python3 -m nose --processes=-1 -vw test/unittests
+container-test:
+	podman build --tag $(CONTAINER_NAME) --file test/Dockerfile
+	podman run --volume .:/kdump-anaconda-addon:Z $(CONTAINER_NAME) make test
+
+test: runpylint unittest
 
 runpylint:
-	@find . -name '*.py' -print|xargs -n1 --max-procs=$(NUM_PROCS) pylint -E 2> /dev/null
```

Comment:
Pylint was unhappy with imports in the unit tests and I couldn't figure out how to set up PYTHONPATH in this line.